AdClickthroughWorkerTests.swift

```diff
@@ -0,0 +1,127 @@
+//  Copyright 2019, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import XCTest
+import PlayerCore
+import SafariServices
+@testable import VerizonVideoPartnerSDK
+
+class AdClickthroughWorkerTests: XCTestCase {
+    let recorder = Recorder()
+    var showSafariHook: (() -> ())!
```

Comment:
I don't think we need to store hooks on level of class - local scope will be enough.